scripts/configs.js

```diff
@@ -99,10 +98,8 @@ function genConfig(options) {
 
   config.plugins.push(replace(vars))
 
-  if (options.transpile !== false) {
-    config.plugins.push(babel())
-    config.plugins.push(buble())
-  }
+  config.plugins.push(babel())
+  config.plugins.push(buble())
```

Comment:
I'm not sure about Bublé here. Why to we need it in addition to babel ? In my mind `@babel/preset-env` was sufficient to output ES5 code